src/datasets/features/audio.py

```diff
@@ -92,7 +94,23 @@ def encode_example(self, value: Union[str, dict]) -> dict:
             return {"bytes": buffer.getvalue(), "path": None}
         elif value.get("path") is not None and os.path.isfile(value["path"]):
             # we set "bytes": None to not duplicate the data if they're already available locally
-            return {"bytes": None, "path": value.get("path")}
+            if value.get("path").endswith("pcm"):
+                # "PCM" is only have byte RAW-Audio data style.
+                if value.get("sampling_rate") is None:
+                    # At least, If you want to convert "PCM-byte" to "WAV-byte", you have to know sampling rate
+                    raise KeyError("If you want to use pcm, you have to know your 'sampling_rate'!")
```

Comment:
small suggestion for the error message:
```suggestion
                    raise KeyError("To use PCM files, please specify a 'sampling_rate' in addition to the 'path'.")
```